projects/components/src/paginator/paginator.component.ts

```diff
@@ -115,6 +119,8 @@ export class PaginatorComponent implements OnChanges, PaginationProvider {
   // Caused either by a change in the provided page, or user change being emitted
   public readonly pageEvent$: Observable<PageEvent> = merge(this.pageChange, this.pageSizeInputSubject);
 
+  public readonly minItemsBeforeDisplay: number = 10;
```

Comment:
I think this should be configurable (by the parent component), but I don't want to expose it with `@Input()` and then be faced with the prospect of making breaking changes later on. Also, we may be able to come up with a better name if we are to expose this to the parent.

Reply:
That's fair. Let's wait for our first use case where we want to modify it, then we can cross that bridge.